### PR TITLE
Dilpreet | Test | Pull from PR #9919 and resolved conflicts

### PR DESCRIPTION
**Pull request by dilpreet77**
https://app.clickup.com/t/22xahrh

### CODE REVIEW

File: apps/web-giddh/src/app/userDetails/components/subscriptions-plans/subscriptions-plans.component.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { AuthenticationService } from '../../../services/authentication.service'
 import { AppState } from '../../../store';
 import { SettingsProfileActions } from '../../../actions/settings/profile/settings.profile.action';
 import { CompanyActions } from '../../../actions/company.actions';
-import { Router, ActivatedRoute } from '@angular/router';
+import { Router, ActivatedRoute, NavigationStart} from '@angular/router';
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/userDetails/components/subscriptions-plans/subscriptions-plans.component.ts:12]: Unused import

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 2]
```diff
@@ -6087,9 +6087,6 @@ highcharts-chart {
 .panel {
     border-radius: 2px;
 }
-.invoice-modal-content .app-aside-menu-other-taxes:host {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss:6090]: Please check this class is getting removed

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 2 above)
Comment (dilpreet77) [reply apps/web-giddh/src/assets/css/style-2.scss:6090]: checked

File: apps/web-giddh/src/app/userDetails/components/subscriptions-plans/subscriptions-plans.component.ts (hunk 1 above)
Comment (dilpreet77) [reply apps/web-giddh/src/app/userDetails/components/subscriptions-plans/subscriptions-plans.component.ts:12]: resolved

**Review by prateek-giddh — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**